src/documents/signals/handlers.py

```diff
@@ -73,3 +73,13 @@ def run_external_script(sender, document, **kwargs):
         str(document.correspondent),
         str(",".join(document.tags.all().values_list("slug", flat=True)))
     )).wait()
+
+def run_pre_consume_external_script(sender, filename, **kwargs):
```

Comment:
1. Needs an additional blank line before to conform to PEP8.
2. Maybe rename to `run_pre_consume_script` (see [other comment](https://github.com/danielquinn/paperless/pull/127/files#r68360479))